### ISSUE TITLE
"My reviews" contain {{item.title}} item

### ISSUE THREAD

**Issue by yurydelendik** [Reviewable/Reviewable#107]
(.. and probably second issue: duplicate entries appear when the page kept opened for long time)

![screen shot 2015-06-01 at 5 34 28 pm](https://cloud.githubusercontent.com/assets/1523410/7924881/aa3ca17c-0884-11e5-8028-4dc9590db8b0.png)

**Comment by pkaminski** [comment]
Hey @yurydelendik, that looks pretty bad, sorry.  However, I keep my own Reviews page open all the time for days on end and have never observed this behavior so there's probably something else at play here.  Can you provide some more details?
- Does the page appear correct (no unexpanded templates, no duplicates) when first loaded?
- When do the duplicates / templates appear?  (The page refreshes the list of PRs every 5 minutes after loading.)
- Is there anything showing in the console?
- Which browser are you using?

Thanks!

**Comment by yurydelendik** [comment]
> Does the page appear correct (no unexpanded templates, no duplicates) when first loaded?
> When do the duplicates / templates appear?

I see ` {{item.title}}` first time page loaded, but not duplicates

> Is there anything showing in the console?

I see the message below with every duplicate:
```
Exception { message: "", result: 2152398858, name: "NS_ERROR_MALFORMED_URI", filename: "https://reviewable.io/js/f9088ca3.l…", lineNumber: 14, columnNumber: 0, inner: null, data: null, stack: "[3]</m.prototype.end@https://review…" } undefined
```

> Which browser are you using?

Firefox Nightly (41)

**Comment by pkaminski** [comment]
Are you by any chance running in private mode, or otherwise filtering your network connections?

**Comment by yurydelendik** [comment]
The same results running in fresh profile. mozregression points to changes in Firefox 39 (https://hg.mozilla.org/mozilla-central/pushloghtml?fromchange=840cfd5bc971&tochange=db0409de517a)

**Comment by pkaminski** [comment]
I'm unable to reproduce this with FF 39 or FF 41.0a1 (2015-06-01) on Windows, in either normal or private mode, or with FF Nightly on Mac OS X in normal mode.  (We did get FF Nightly to crash completely in private mode on OS X, but that's likely a separate issue.)  Can you provide any more details on your environment?

At this point, I'm guessing that the `NS_ERROR_MALFORMED_URI` error is the culprit, stopping the script and preventing the templates from being evaluated.  I have no idea what might be causing it, though.

**Comment by yurydelendik** [comment]
Per commit bisect points to bug 1144398

```
The first bad revision is:
changeset:   235138:c334736aa584
user:        Valentin Gosu <valentin.gosu@gmail.com>
date:        Wed Mar 18 18:40:35 2015 +0200
summary:     Bug 1144398 - Restrict charcters which are allowed in the URL hash (ref). r=mcmanus
```

**Comment by pkaminski** [comment]
Thanks!  Unfortunately I'm not authorized to access that bug.  :disappointed:  Are you able to tell me what characters are restricted from appearing in the hash?  Though the Reviews page doesn't make use of the hash at all, so I don't yet see how it could be affected.

**Comment by yurydelendik** [comment]
Looks like it fails only on certain PRs that's why you are unable to reproduce.

I don't have access to the bug itself, but looking and tweaking the commit I got invalid URL (verbatim below):

```
https://api.github.com/repos/mozilla/pdf.js/pulls/4813?per_page=100&Even though this should work as requested, there might still be room for improvements of the code.
Fixes #4810.

Before we accept this PR, there are two questions that I believe that we need to ask ourselves:
* First of all, do we actually want this feature?
* Secondly, does this work properly on touch?

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/mozilla/pdf.js/4813)
<!-- Reviewable:end -->

```

**Comment by pkaminski** [comment]
Wow, that was an epic fail on my part.  Thank you for spotting it and tracking it down -- should be fixed now.
